content/influxdb/v2.0/query-data/influxql.md

```diff
@@ -43,13 +43,48 @@ By default, the `/query` compatibility endpoint returns results in **JSON**.
 To return results as **CSV**, include the `Accept: application/csv` header.
 
 ## Database and retention policy mapping
+
 InfluxDB 2.0 combines the 1.x concept of [databases](/influxdb/v1.8/concepts/glossary/#database)
 and [retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp)
 into [buckets](/influxdb/v2.0/reference/glossary/#bucket).
 To support InfluxDB 1.x query and write patterns in InfluxDB 2.0, databases and retention
 policies are mapped to buckets using the **database and retention policy (DBRP) mapping service**.
 _See [DBRP mapping](/influxdb/v2.0/reference/api/influxdb-1x/dbrp/) for more information._
 
+### Map a bucket to a database and retention policy
+
+If you have an existing bucket that does't follow the **database/retention-policy**
+naming convention, you **must** manually create a database and retention policy
+mapping to query that bucket with the `/query` compatibility API.
+Use the using the [`/api/v2/dbrps` API endpoint](/influxdb/v2.0/api/#tag/DBRPs) to
+manually create and manage DBRP mappings.
```

Comment:
So, the first question is how would the end user know if the existing bucket has a database/retention-policy naming convention or an existing mapping?  

There is some additional context missing here which might be valuable.

The mapping is automatically created if and only if the 1.x write compatibility API is used.

If the user has used the 2.x write API to feed data into InfluxDB AND wishes to access that data using InfluxQL, a mapping is required.

Reply:
there isn't anywhere to see this mapping in the UI.  Which means they probably need to run the CURL command to list the existing mappings to review/confirm...   see the API reference docs for the appropriate params to construct that query.  It's very similar to the below.

Reply:
https://docs.influxdata.com/influxdb/v2.0/api/#operation/GetDBRPs